app/controllers/crime_applications_controller.rb

```diff
@@ -5,4 +5,23 @@ def index
   end
 
   def edit; end
+
+  def destroy
+    @application = CrimeApplication.find(delete_params[:id])
+    @full_name = @application.applicant.full_name
+
+    if request.get?
+      render :delete
+    elsif request.delete?
+      @application.destroy
+      flash[:alert] = "#{@full_name}'s application has been permenantly deleted"
```

Comment:
i18n? and a little typo: `permenantly` -> `permanently`

We can do the flash and redirect in one go with:

```ruby
redirect_to crime_applications_path, flash: { alert: 'message' }
```